lib/ansible/modules/web_infrastructure/ansible_tower/tower_inventory.py

```diff
@@ -37,6 +37,13 @@
     variables:
       description:
         - Inventory variables. Use C(@) to get from file.
+    kind:
```

Comment:
These need `version_added: "2.6"` values.

Reply:
Added, thank you!